src/index.html

```diff
@@ -286,17 +321,34 @@
         });
     };
 
+    // As the map moves, grab and update bounds in inset map.
+    if (config.inset) {
+    map.on('move', getInsetBounds);
+    }
     // setup the instance, pass callback functions
     scroller
     .setup({
         step: '.step',
         offset: 0.5,
         progress: true
     })
-    .onStepEnter(response => {
+    .onStepEnter(async response => {
         var chapter = config.chapters.find(chap => chap.id === response.element.id);
         response.element.classList.add('active');
-        map[chapter.mapAnimation || 'flyTo'](chapter.location);
+        map[chapter.mapAnimation || 'flyTo'](
+            {...chapter.location, preload: true}
```

Comment:
Ref https://github.com/mapbox/mapbox-gl-js/pull/11328

Reply:
I'm still noticing unloaded tiles in the template:

https://user-images.githubusercontent.com/12721145/157057547-6dae2456-aff5-440d-9f7a-2da0a50c89b2.mp4

Reply:
```
map[chapter.mapAnimation || 'flyTo'](
            {...chapter.location, preloadOnly: true}
        );
await map.once('idle');
map[chapter.mapAnimation || 'flyTo'](
            {...chapter.location}
        );
```



I was able to get it working with the `preloadOnly` flag, but I am now a little skeptical of it. For a style like `satellite-street`, this flag adds a significant lag as the map waits for entire tiles in the path to be loaded before starting the transition. 

I think I am going to drop this flag for now and maybe add it when the flag goes through a few more version iterations.